src/conversations/Conversations.ts

```diff
@@ -152,6 +160,38 @@ export default class Conversations<ContentTypes = any> {
     return this.decodeInvites(envelopes)
   }
 
+  private async validateConsentSignature(
+    signature: `0x${string}`,
+    timestamp: number,
```

Comment:
What scale is timestamp at? Usually we try and add the `_ns` suffix to avoid confusion. And if it's nanoseconds, you're better off treating it as a Long, since 64 bit ints can overflow a Number and throw weird errors if someone sneaks a big one in.